### ISSUE TITLE
GRIM: Don's computer isn't properly displayed with the TinyGL driver

### ISSUE THREAD

**Issue by eriktorbjorn** [residualvm/residualvm#1114]
When using the TinyGL driver, Don's computer isn't properly displayed. The selected text does change color, but its background doesn't change. And instead of the effect simulating a bad monitor, the text just flickers.

This used to work in 1.1.0, but that was quite some time ago. I tried bisecting, but there are an annoying number of revisions where the TinyGL renderer doesn't work or crashes when trying to bring up the save/load screen. My attempt did eventually bring up this:

```
commit 7f560daf0475bab1e387f39daa47ce0acbb58b19
Author: Stefano Musumeci <subr3v@gmail.com>
Date:   Sat Jul 26 17:56:44 2014 +0200

    TINYGL: Replaced Array with List as the container for draw calls queue.
    NOTE: I also had to construct the struct GLContext properly now.

:040000 040000 bdd7742eff28d22ff95de8fd9aca96d24c73dbd6 d30f3124448178e12bfaf55c6f1bea1131fea88b M  graphics
```

But I haven't yet verified that this actually _is_ the offending commit.

**Comment by vpelletier** [comment]
Looking into this, I also checked other calls to other drawing lua_v1_graphics.cpp functions (DrawPolygon, DrawLine, DrawRectangle).
I found that DrawLine, used in set "tp" (ticket printer) to draw numbers & letters, also doesn't work with TinyGL. This is a game-breaker bug.

**Comment by vpelletier** [comment]
This should be fixed since 4c41a1e4a577ffc1c31ec4411ff302281d8feac6 (July 2016) as these drawings are no longer done directly on the frame buffer. The same issue was affecting the ticket printer's digits.
